src/doc/src/reference/source-replacement.md

```diff
@@ -87,13 +97,13 @@ git = "https://example.com/path/to/repo"
 
 ## Registry Sources
 
-A "registry source" is one that is the same as crates.io itself. That is, it has
-an index served in a git repository which matches the format of the
-[crates.io index](https://github.com/rust-lang/crates.io-index). That repository
-then has configuration indicating where to download crates from.
+A "registry source" is one that works like crates.io itself. It's an index
+which matches the format of the [crates.io index] with a configuration file
+indicating where to download crates from.
+
+Registry sources can use [either git or sparse HTTP protocol][protocols].
 
-Currently there is not an already-available project for setting up a mirror of
-crates.io. Stay tuned though!
+[crates.io index]: https://github.com/rust-lang/crates.io-index
```

Comment:
If we're linking out for the index, we should probably link out to https://doc.rust-lang.org/cargo/reference/registry-index.html